### DISCUSSION TITLE
Eksporter

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#24] [Questions] [answered]
Vi havde en god og afklarende snak til statusmødet i går om dashboard, lister og eksporter (jeg blev klogere, i hvert fald). Jeg har et fint billede af dashboard og lister. 

Men hvad med eksport, som jo nævnes flere gange i acceptkriterierne?

Er der måske følgende?

Person (for- og efternavn(e)) | Frivillig
Email | Frivillig
Telefon | Frivillig
Bopæl (postnummer) | Frivillig
Fødselsdag (alder i år) | Frivillig
Køn | Frivillig
T-shirt størrelse | Frivillig
Nationalitet | Frivillig
Primært sprog | Frivillig
Taler (Sprogkundskab) | Frivillig
Lables | Frivillig
Medlemsskab (oprettelsesdato) | Frivillig
Sidst set (seneste login) | Frivillig
Tilmeldte aktuelle vagter | Frivillig
Gennemførte vagter | Frivillig
Frivilligtimer | Frivillig
Org navn | Arrangør
Adresse (gade, hus nr. osv) | Arrangør
Postnummer | Arrangør
Hovedarrangør, navn | Arrangør
Hovedarrangør, mail | Arrangør
Hovedarrangør, tlf | Arrangør
CVR eller navn på privatperson | Arrangør
NY: Antal aktuelle events | Arrangør
NY: Antal tidligere events | Arrangør
NY: Antal frivilligtimer modtaget | Arrangør
Eventnavn | Event
Vagter | Event
Eventets arrangør | Event
Eventsted (postnummer) | Event
Antal vagtdage | Event
Antal vagter udbudt | Event
Antal vagter taget af frivillige | Event
Antal vagttimer (udbudte timer) | Event
Antal frivilligtimer (matchede timer) | Event

Jeg har samlet listen her: https://docs.google.com/spreadsheets/d/1xocGICUCHaEGjVw1L8CM6EqDJ-HtJnThWa0K1uIHVzk/edit?gid=1400388749#gid=1400388749

Derudover er der diplomet, som jeg vender tilbage med et oplæg til, når jeg har afklaret det med Rethinker.

**Comment by mikkelsl** [comment, marked as answer]
Vi kan bare tage hvert acceptkriterie enkeltvis, det gør det nemmere at forholde sig til den enkelte opgave. Men eksport af lister er ikke noget problem at lave og nemt at løse.

Fint med en god skabelon til diplom.
